tpl/config-wizard.html.twig

```diff
@@ -281,7 +281,7 @@ ciphers ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES2
 		</tr>
 		<tr class="tab_bg_1">
 			<td class="right" colspan="2">
-			{{ __('Wizard page:','flyvemdm') }}&nbsp;{{ step }}
+			{{ __('Wizard page:','flyvemdm') }}&nbsp;{{ step }} / {{total_step}}
```

Comment:
This number is a technical information to more easily debug the wizard.

Reply:
I agree with you, but going from step 1 to step 100 is confusing for users

Reply:
how do you imagine this part

Reply:
We may just change the label to tell this is a technical ID ?

Reply:
Done

Reply:
Wizard page is renamed to Technical step ID